health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/configs/UICustomizations.js

```
@@ -218,33 +219,54 @@ export const UICustomizations = {
             return res;
           }
           const delay = (ms) => new Promise(resolve => setTimeout(resolve, ms));
+          const [showToast, setShowToast] = useState(null);
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)